src/toil/jobStores/abstractJobStore.py

```diff
@@ -422,6 +422,28 @@ def updateFileStream(self, jobStoreFileID):
         """
         raise NotImplementedError()
 
+    @abstractmethod
+    def importFile(self, sourceUrl):
+        """
+        Imports file pointed at by sourceUrl into filesBucket of jobstore. The jobStoreFileId of
+        the new file is returned.
+
+        :param sourceUrl: URL of scheme 'file:' pointing to local files and urls pointing to the
+        local file scheme of the jobstore, e.g. 's3:' for AWS
```

Comment:
I think that Sphinx will complain about this docstring when we build the docs. If the :param description contains multiple lines it should have a \ to delimitate each line. To confirm this, you can run 'make docs' and see what output is generated. See the Job class for examples on properly formatted multiline descriptions.